apps/mochi-web/components/Profile/ProfileWidget.tsx

```diff
@@ -210,9 +207,15 @@ export const ProfileWidget = () => {
       </div>
       <Tabs value={selectedChain}>
         <TabList
-          className="flex items-center -mx-4 border-t border-b border-divider"
+          className="flex items-center -mx-4 border-t border-b border-divider relative"
           ref={tabElement}
         >
+          <div
+            className="bg-background-level2 absolute top-0 left-0 h-full w-[120px] transition-transform duration-300"
+            style={{
+              transform: `translateX(${translateX}px)`,
+            }}
+          />
```

Comment:
This appears to be broken on my account:

<img width="598" alt="image" src="https://github.com/consolelabs/mochi-ui/assets/12707960/c44d5322-8744-4a8c-95d1-29831175a592">